<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xxhè§†é¢‘å‹åˆ¶å·¥å…· v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="main_layout" stretch="1,2">
    <item>
     <layout class="QVBoxLayout" name="left_panel">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text"><string>&lt;b&gt;ğŸ“ åŒºåŸŸä¸€ï¼šæ–‡ä»¶è°ƒåº¦&lt;/b&gt;</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_input">
        <property name="text"><string>é€‰æ‹©åŸè§†é¢‘</string></property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_input">
        <property name="placeholderText"><string>ç­‰å¾…å¯¼å…¥...</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_output">
        <property name="text"><string>è®¾ç½®å¯¼å‡ºè·¯å¾„</string></property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="output_layout">
        <item>
         <widget class="QLineEdit" name="txt_output">
          <property name="placeholderText"><string>ç­‰å¾…è®¾ç½®...</string></property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_format">
          <property name="minimumSize"><size><width>75</width><height>0</height></size></property>
          <property name="styleSheet"><string>font-weight: bold;</string></property>
          <item><property name="text"><string>.mp4</string></property></item>
          <item><property name="text"><string>.mkv</string></property></item>
          <item><property name="text"><string>.mov</string></property></item>
          <item><property name="text"><string>.flv</string></property></item>
          <item><property name="text"><string>.avi</string></property></item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_1">
        <property name="orientation"><enum>Qt::Vertical</enum></property>
        <property name="sizeType"><enum>QSizePolicy::Fixed</enum></property>
        <property name="sizeHint"><size><width>20</width><height>20</height></size></property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text"><string>&lt;b&gt;âš™ï¸ åŒºåŸŸäºŒï¼šå‹åˆ¶ç­–ç•¥&lt;/b&gt;</string></property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_preset"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_preview">
        <property name="text"><string>å¼€å¯å®æ—¶ç”»é¢é¢„è§ˆ</string></property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tab_custom">
        <property name="visible"><bool>false</bool></property>
        <property name="currentIndex"><number>0</number></property>
        <widget class="QWidget" name="tab_video">
         <attribute name="title"><string>è§†é¢‘è®¾ç½®</string></attribute>
         <layout class="QFormLayout" name="layout_video">
          <item row="0" column="0"><widget class="QLabel" name="lbl_v1"><property name="text"><string>ç¼–ç å™¨:</string></property></widget></item>
          <item row="0" column="1"><widget class="QComboBox" name="cb_v_encoder"/></item>
          <item row="1" column="0"><widget class="QLabel" name="lbl_v2"><property name="text"><string>å¸§ç‡(FPS):</string></property></widget></item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cb_v_fps">
            <item><property name="text"><string>ä¿æŒæº</string></property></item>
            <item><property name="text"><string>24</string></property></item>
            <item><property name="text"><string>30</string></property></item>
            <item><property name="text"><string>60</string></property></item>
           </widget>
          </item>
          <item row="2" column="0"><widget class="QLabel" name="lbl_v3"><property name="text"><string>åˆ†è¾¨ç‡:</string></property></widget></item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cb_v_res">
            <item><property name="text"><string>ä¿æŒæº</string></property></item>
            <item><property name="text"><string>720p</string></property></item>
            <item><property name="text"><string>1080p</string></property></item>
            <item><property name="text"><string>1440p</string></property></item>
            <item><property name="text"><string>2160p</string></property></item>
           </widget>
          </item>
          <item row="3" column="0"><widget class="QLabel" name="lbl_v4"><property name="text"><string>ç ç‡æ§åˆ¶:</string></property></widget></item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cb_v_rc">
            <item><property name="text"><string>cqp</string></property></item>
            <item><property name="text"><string>vbr</string></property></item>
            <item><property name="text"><string>cbr</string></property></item>
           </widget>
          </item>
          <item row="4" column="0"><widget class="QLabel" name="lbl_v5"><property name="text"><string>å‚æ•°æ•°å€¼:</string></property></widget></item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="val_layout">
            <item>
             <widget class="QSlider" name="sld_v_value">
              <property name="orientation"><enum>Qt::Horizontal</enum></property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_v_val_display">
              <property name="minimumSize"><size><width>60</width><height>0</height></size></property>
              <property name="styleSheet"><string>font-weight: bold; color: #225555;</string></property>
              <property name="text"><string>32</string></property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_audio">
         <attribute name="title"><string>éŸ³é¢‘è®¾ç½®</string></attribute>
         <layout class="QFormLayout" name="layout_audio">
          <item row="0" column="0"><widget class="QLabel" name="lbl_a1"><property name="text"><string>ç¼–ç å™¨:</string></property></widget></item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cb_a_encoder">
            <item><property name="text"><string>aac</string></property></item>
            <item><property name="text"><string>mp3</string></property></item>
            <item><property name="text"><string>copy</string></property></item>
            <item><property name="text"><string>an (å‰¥ç¦»é™éŸ³)</string></property></item>
           </widget>
          </item>
          <item row="1" column="0"><widget class="QLabel" name="lbl_a2"><property name="text"><string>ç ç‡:</string></property></widget></item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cb_a_bitrate">
            <item><property name="text"><string>320k</string></property></item>
            <item><property name="text"><string>192k</string></property></item>
            <item><property name="text"><string>128k</string></property></item>
            <item><property name="text"><string>96k</string></property></item>
            <item><property name="text"><string>64k</string></property></item>
           </widget>
          </item>
          <item row="2" column="0"><widget class="QLabel" name="lbl_a3"><property name="text"><string>é‡‡æ ·ç‡:</string></property></widget></item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cb_a_sample">
            <item><property name="text"><string>ä¿æŒæº</string></property></item>
            <item><property name="text"><string>44100</string></property></item>
            <item><property name="text"><string>48000</string></property></item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation"><enum>Qt::Vertical</enum></property>
        <property name="sizeHint"><size><width>20</width><height>40</height></size></property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_start">
        <property name="minimumSize"><size><width>0</width><height>45</height></size></property>
        <property name="styleSheet"><string>font-weight: bold; font-size: 14px;</string></property>
        <property name="text"><string>ğŸš€ å¼€å§‹å‹åˆ¶</string></property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="btn_layout">
        <item>
         <widget class="QPushButton" name="btn_pause">
          <property name="enabled"><bool>false</bool></property>
          <property name="text"><string>â¸ æš‚åœ</string></property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_stop">
          <property name="enabled"><bool>false</bool></property>
          <property name="text"><string>â¹ åœæ­¢</string></property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="right_panel">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text"><string>&lt;b&gt;ğŸ“º åŒºåŸŸä¸‰ï¼šå®æ—¶ç›‘æ§å±&lt;/b&gt;</string></property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_preview">
        <property name="minimumSize"><size><width>480</width><height>270</height></size></property>
        <property name="styleSheet"><string>background-color: #1e1e1e; color: #888888; border-radius: 8px; font-size: 16px;</string></property>
        <property name="text"><string>ç”»é¢é¢„è§ˆåŒº
(ç­‰å¾…å‹åˆ¶å¼€å§‹...)</string></property>
        <property name="alignment"><set>Qt::AlignCenter</set></property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation"><enum>Qt::Vertical</enum></property>
        <property name="sizeType"><enum>QSizePolicy::Fixed</enum></property>
        <property name="sizeHint"><size><width>20</width><height>10</height></size></property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text"><string>&lt;b&gt;ğŸ“Š åŒºåŸŸå››ï¼šè¿è¡ŒçŠ¶æ€&lt;/b&gt;</string></property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progress_bar">
        <property name="value"><number>0</number></property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_status">
        <property name="styleSheet"><string>color: #666666;</string></property>
        <property name="text"><string>çŠ¶æ€: é—²ç½® | é€Ÿåº¦: -- | å‰©ä½™æ—¶é—´: -- |  å½“å‰æ–‡ä»¶å¤§å°: --</string></property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation"><enum>Qt::Vertical</enum></property>
        <property name="sizeHint"><size><width>20</width><height>40</height></size></property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>